pyglet/window/__init__.py

```diff
@@ -589,7 +594,7 @@ def __init__(self,
             screen = display.get_default_screen()
 
         if not config:
-            for template_config in [gl.Config(double_buffer=True, depth_size=24),
+            for template_config in [gl.Config(double_buffer=True, depth_size=24, alpha_size=8),
```

Comment:
There is a catch-all for the alpha size later, so we probably don't want to enforce that here.

Reply:
Hmm... Running locally, if I dont have this here (or specify explicitly in my config) it does _not_ work for me.